es-design-system/pages/organisms/es-reviews-list.vue

```diff
@@ -162,12 +133,18 @@ export default {
 @import '~@energysage/es-bs-base/scss/includes';
 
 .review-wrapper {
-    width: 450px;
+    border-right: 1px solid $border-color;
+    width: 470px;
```

Comment:
On my screen, the third review is scrunched to the side (screenshot below). I would expect that the width of each of the reviews would be responsive to viewport width. 
<img width="1344" alt="Screen Shot 2022-12-09 at 3 49 10 PM" src="https://user-images.githubusercontent.com/91746907/206793731-b905109f-75f4-4ec0-abac-fd5c98b634e7.png">

This change worked for me, but I'm not sure if that's exactly what we're going for 🤷‍♀️ 
```suggestion
    width: 30%;
```

Reply:
Good call out. Looking at it more there's no reason this should just be a `b-row` b-col` layout now. I've updated the docs.